.github/CODEOWNERS

```diff
@@ -71,8 +71,8 @@ exporter/lokiexporter/                            @open-telemetry/collector-cont
 exporter/mezmoexporter/                           @open-telemetry/collector-contrib-approvers @dashpole @billmeyer @gjanco
 exporter/opencensusexporter/                      @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
 exporter/otelarrowexporter/                       @open-telemetry/collector-contrib-approvers @jmacd @moh-osman3 @lquerel
-exporter/prometheusexporter/                      @open-telemetry/collector-contrib-approvers @Aneurysm9 @dashpole
-exporter/prometheusremotewriteexporter/           @open-telemetry/collector-contrib-approvers @Aneurysm9 @rapphil @dashpole
+exporter/prometheusexporter/                      @open-telemetry/collector-contrib-approvers @Aneurysm9 @dashpole @ArthurSens
```

Comment:
Please update https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusexporter/metadata.yaml#L11 too, same for other packages

Reply:
yep, forgot.  Running it now